azurerm/internal/services/lighthouse/lighthouse_definition_resource_test.go

```diff
@@ -155,10 +180,13 @@ data "azurerm_role_definition" "contributor" {
   role_definition_id = "b24988ac-6180-42a0-ab88-20f7382dd24c"
 }
 
+data "azurerm_subscription" "test" {}
```

Comment:
Is there any particular reason to modify the existing tests?

Reply:
the exsiting test case would fail because of missing a required property